devicetypes/smartthings/zwave-switch-generic.src/zwave-switch-generic.groovy

```diff
@@ -185,7 +185,7 @@ def ping() {
 
 def refresh() {
 	def commands = []
-	commands << zwave.basicV1.basicGet().format()
+	commands << zwave.switchBinaryV1.switchBinaryGet().format()
```

Comment:
The device didn't respond to the basic get?

Reply:
It didn't:

```
2019-03-12T13:17:36.224+0100 |  2372 | I | zw         | Node [0F] acked 20 02 
2019-03-12T13:18:49.574+0100 |  2372 | I | zw         | Node [0F] sending 15 min poll report: 0
2019-03-12T13:19:31.155+0100 |  2372 | I | zw         | Polled switch [0F] changed from 0 to 255
2019-03-12T13:19:31.247+0100 |  2372 | I | zw         | [0F] SwitchBinaryReport(255)
2019-03-12T13:20:55.147+0100 |  2372 | I | zw         | Polled switch [0F] changed from 255 to 0
2019-03-12T13:20:55.164+0100 |  2372 | I | zw         | [0F] SwitchBinaryReport(0)
2019-03-12T13:21:46.153+0100 |  2372 | I | zw         | Node [0F] acked 20 02 
2019-03-12T13:33:49.780+0100 |  2372 | I | zw         | Node [0F] sending 15 min poll report: 0
2019-03-12T13:48:49.985+0100 |  2372 | I | zw         | Node [0F] sending 15 min poll report: 0
2019-03-12T14:03:59.956+0100 |  2372 | I | zw         | Node [0F] sending 15 min poll report: 0
```

This is just a guess, though. I don't have this device.

Reply:
`20 02` is a basic get

I'm pretty sure the polling uses switchBinary.

Reply:
not trying to commit this rn, just wanted a PR for SR-Pol to try with.

Reply:
`sending 15 min poll reports` means that the device is responding to Basic Gets from the local polling module

Reply:
Yep. You're right. Mistakenly assumed the polling code used binary switch.

Still not sure why the cloud basicGet isn't being responded to, though.

Afaik this isn't a secure device. It doesn't even report that it supports security so I'm a bit baffled:

https://products.z-wavealliance.org/products/1945

Only other guess is just interference?

Reply:
Was the device actually polled by the cloud ever though? If local polling works (Uses Basic Get), then the cloud would not ping the device via Switch Binary Get.

Can the testers provide info how often this issue happens with Basic Get versus Switch Binary Get? If we assume this is the change that we need to make, then we are essentially stating the device may not respond to Basic Gets. I believe that this assumption is not the case and the issue is else where, but that is based on gut feeling and not understanding why a device would choose to ignore a Basic Get.

Reply:
I had them test by manually hitting "refresh" which would have sent a basic get. There's a screenshot on the ticket of the device not responding in the IDE. Frankly, I don't get it either. Because if it's interference the polls shouldn't be getting through reliably when the pings don't.

Reply:
Do we have this device in MV office? Can I get one? I would like to investigate

Reply:
We've got one in MV. I'll try to poke at it some tomorrow.